app/Http/Controllers/CarrinhoController.php

```
@@ -2,63 +2,300 @@
+        $item->fill($request->only(['preco', 'quantidade']));
+        $item->save();
+
+        return response()->json($item);
     }
 
-    /**
-     * Update the specified resource in storage.
-     */
-    public function update(Request $request, string $id)
+    // API: remover foto do carrinho por id
+    public function removeFoto(Request $request, $id, $fid)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)